Looks like there are a couple unit tests failing: 
```
======================================================================
FAIL: testDownloadConfig (api.api_server_test.testTurbiniaAPIServer.testDownloadConfig)
Test downloading current Turbinia server config.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turbinia/turbinia/api/api_server_test.py", line 236, in testDownloadConfig
    self.assertEqual(response.text, content)
AssertionError: '{"INSTANCE_ID":"turbinia-instance1","CLOU[4117 chars]ai"}' != '# -*- coding: utf-8 -*-\n# Copyright 2016[15437 chars]\'\n'
Diff is 19632 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_update_rules (workers.analysis.yara_test.YaraAnalysisTaskTest.test_update_rules)
Tests the update_rules method
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turbinia/turbinia/workers/analysis/yara_test.py", line 81, in test_update_rules
    self.assertEqual(ret, True)
AssertionError: False != True

----------------------------------------------------------------------
````

--- a/turbinia/api/api_server_test.py
+++ b/turbinia/api/api_server_test.py
@@ -228,12 +228,10 @@ class testTurbiniaAPIServer(unittest.TestCase):
     self.assertEqual(response.json(), config_dict)
 
   def testDownloadConfig(self):
-    """Test downloading current Turbinia server config."""
-    with open(turbinia_config.CONFIG.__file__, mode='r',
-              encoding='utf-8') as file:
-      content = ''.join(file.readlines())
+    """Test downloading current Turbinia server config (redacted)."""
+    config_dict = turbinia_config.toDict()
     response = self.client.get('/api/config/download')
-    self.assertEqual(response.text, content)
+    self.assertEqual(response.json(), config_dict)
 
   def testRequestResultsNotFound(self):
     """Test getting empty request result files."""

--- a/turbinia/config/__init__.py
+++ b/turbinia/config/__init__.py
@@ -249,10 +249,6 @@ def ParseDependencies():
 SENSITIVE_VARS = frozenset([
     'EMAIL_PASSWORD',
     'GCP_GENERATIVE_LANGUAGE_API_KEY',
-    'CELERY_BROKER',
-    'CELERY_BACKEND',
-    'KOMBU_BROKER',
-    'DFDEWEY_OS_URL',
     'OIDC_KEYS',
     'WEBUI_CLIENT_SECRETS_FILE',
 ])

